pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -32,48 +50,43 @@ import (
 // deployments will be cancelled. The controller will not attempt to scale
 // running deployments.
 type DeploymentConfigController struct {
-	// kubeClient provides acceess to Kube resources.
-	kubeClient kclient.Interface
-	// osClient provides access to OpenShift resources.
-	osClient osclient.Interface
+	// oc provides access to OpenShift resources.
+	oc osclient.Interface
+	// kc provides acceess to Kube resources.
+	kc kclient.Interface
+
+	// queue contains deployment configs that need to be synced.
+	queue *workqueue.Type
+
+	// dcStore provides a local cache for deploymentconfigs.
+	dcStore oscache.StoreToDeploymentConfigLister
+	// Watches changes to all deployments
+	dcController *framework.Controller
+	// A store of replication controllers, populated by the rcController.
+	rcStore cache.StoreToReplicationControllerLister
+	// Watches changes to all replication controllers.
+	rcController *framework.Controller
+	// A store of pods, populated by the podController
+	podStore cache.StoreToPodLister
+	// Watches changes to all pods
+	podController *framework.Controller
```

Comment:
I need to know if the pod cache has synced in order to proceed with the dc reconcilation.